<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="MyTest" duration-ms="22" started-at="2017-05-22T02:53:00Z" finished-at="2017-05-22T02:53:00Z">
    <groups>
    </groups>
    <test name="automation" duration-ms="22" started-at="2017-05-22T02:53:00Z" finished-at="2017-05-22T02:53:00Z">
      <class name="com.ZhiHu.autotest.testCase.Something">
        <test-method status="FAIL" signature="setUp()[pri:0, instance:com.ZhiHu.autotest.testCase.Something@72d818d1]" name="setUp" is-config="true" duration-ms="0" started-at="2017-05-22T10:53:00Z" finished-at="2017-05-22T10:53:00Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.ZhiHu.autotest.utilities.XmlParseHandler.getNodeValue(XmlParseHandler.java:38)
	at com.ZhiHu.autotest.testCase.BaseTestCaseForWeb.setUp(BaseTestCaseForWeb.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="SignIn()[pri:2, instance:com.ZhiHu.autotest.testCase.Something@72d818d1]" name="SignIn" duration-ms="0" started-at="2017-05-22T10:53:00Z" finished-at="2017-05-22T10:53:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SignIn -->
        <test-method status="SKIP" signature="teardown()[pri:0, instance:com.ZhiHu.autotest.testCase.Something@72d818d1]" name="teardown" is-config="true" duration-ms="0" started-at="2017-05-22T10:53:00Z" finished-at="2017-05-22T10:53:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.ZhiHu.autotest.testCase.Something -->
    </test> <!-- automation -->
  </suite> <!-- MyTest -->
</testng-results>
